### DISCUSSION TITLE
Writing Keystatic docs... in the open 👀

### DISCUSSION THREAD

**Discussion by simonswiss** [Thinkmill/keystatic#105] [Announcements]
Hey friends!

First of all, thank you for trying out Keystatic so early! ❤️

We're aware there is something critically missing from Keystatic right now: **documentation**. 

This is one of the next things we're working on immediately!

Starting today, I'm going to maintain ~a  [public Notion space](https://www.notion.so/thinkmill/Keystatic-9f141cb1ac3c44b5ad02f1ae9b5543d8) where you can see the docs evolve~.

**Update**: the latest docs now live on https://keystatic.com 💫

---

That comes with the caveat that things might be slightly _chaotic_ at times, but this is the beauty of joining a project within the _early peek_ window!

If you have questions about specific parts you'd like documented, ask away in those GitHub discussions right here. This is helping me figure out what are the early pain points, and I can address those as they come!

Thanks again for being part of this journey with us 🤗

Now let's write some docs! ✍️

**Comment by simonswiss** [comment]
Here's the first guide I wrote today: [Setting up Keystatic with an existing Astro v2 project](https://thinkmill.notion.site/Add-Keystatic-to-an-Astro-Project-312e59b9be1f43d480d3f77638da2910) ✨

**Reply by akoepcke** [reply]
In section **Keystatic Admin UI pages**, there's a sentence:
> Now, we can import that file in our keystatic/[...rest].astro page, and mount the component on the clientside only:
 
I believe that should be `[...params].astro` instead?

**Reply by simonswiss** [reply]
Yes, thank you! We initially started using `...rest` before switching to `...params`, and I got mixed up between both 😅

This is fixed now, thanks for pointing that out! 🙏

**Reply by teinett** [reply]
I started new project Astro + Keystatic, using official docs https://keystatic.com/docs/installation-astro.
I suppose, the project uses markdoc integration https://docs.astro.build/en/guides/integrations-guide/markdoc/, but according to the docs we didn't install it. 
Please, add this process to the docs.

**Reply by simonswiss** [reply]
Hey!

It's in that guide you linked to, but a bit later on, at the "rendering content" part:

https://keystatic.com/docs/installation-astro#astro-s-markdoc-integration

Perhaps we could make a note earlier on that if you want to use Astro's content collections, you also need the Markdoc integration 👍

**Reply by simonswiss** [reply]
@teinett I've [re-visited the Astro integration guide](https://github.com/Thinkmill/keystatic/pull/512/files) to mention the Markdoc integration earlier on 👍

**Comment by jschuur** [comment]
> Here's the first guide I wrote today: [Setting up Keystatic with an existing Astro v2 project](https://thinkmill.notion.site/Add-Keystatic-to-an-Astro-Project-312e59b9be1f43d480d3f77638da2910) ✨

This is super timely for me. Amazing!

**Reply by simonswiss** [reply]
Haha awesome, glad it's useful!

**Comment by simonswiss** [comment]
Update: the latest docs now live on https://keystatic.com/ 💫

**Comment by radenpioneer** [comment]
Would like to help, a little bit!
